readme.md

```diff
@@ -1,212 +1,234 @@
+* **Question:** Another field was updated when you ran this command. Which one was it?
+
+* **Your Answer:**
+
+--- The `updated_at` key/value pair was updated as a result of the update to the document.
+
+* **Question:** Take a look at the terminal window running your server. You are likely getting the following deprecation warning:
+  ```
+  DeprecationWarning: Mongoose: `findOneAndUpdate()` and `findOneAndDelete()` without the `useFindAndModify` option set to false are deprecated. See: https://mongoosejs.com/docs/deprecations.html#-findandmodify-
+  ```
+  Take a look [at this page](https://mongoosejs.com/docs/deprecations#findandmodify) to see how to fix it. Describe the changes that took place.
+
+* **Your Answer:**
+
+---You can resolve this issue by adding `useFindAndModify: true` to your mongoDB connection statement . If i am understanding the Mongoose documentation correctly, this statement is added in order to force Mongoose to use MongoDB driver's `findOneandUpdate()` function instead of MongoDB driver's `findAndModify()` function.  Is that correct?
+
+- [ ] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
+
+---
+
+- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
+
+* **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
+
+* **Your Answer:** 
+
+---`Model.create()` failed to return a response, even though the creation of the document was successful in the DB. I am not entirely sure that i understand the error in the terminal, but it seems as though i am dealing with a promise rejection and would need to use something like `.catch()` to handle with an error message, function, etc. Is it because ``Model.create()` is not returning a query?
+
+- [ ] Modify your `GET /api/series` route so that you can search through the content based off of query parameters. For example, if your request was `GET /api/series?start_year=1997`, it would return all series that start in 1997.
+
+---
+
+- [ ] At the moment, there is no validation set on creating any series. Add validation so that each of the three fields are required in order to post to series. Handle any errors that arise with a status code 400 and a brief, standardized message.
+
+* **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
+
+* **Your Answer:**
+
+--- I see the specific messaging coming back in the terminal, but i'm unclear on how to take the `message` key/value pair and parse it to my error message.

```

Comment:
I believe the error should have a `messages` key that is an array of all the errors that occurred. Looping through this will give you access to the individual messages.